src/client/src/rt-styleguide/components/Dropdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons'
+import Button from './Button'
+
+export interface DropdownStyleProps {
+  active?: boolean
+  disabled?: boolean
+  title?: string
+}
+
+export const Dropdown: React.FC<DropdownStyleProps> = props => {
+  const { title, ...rest } = props
+
+  return (
+    <Button {...rest} intent="mute">
+      {`${title} `}
+      <FontAwesomeIcon icon={faAngleDown} />
```

Comment:
<table>
<tr>
<td>Implementation</td><td>Design</td>
</tr>
<tr>
<td>
<img width="205" alt="Screenshot 2020-01-30 at 09 44 59" src="https://user-images.githubusercontent.com/156303/73437968-31786680-4345-11ea-87b2-31a2bfab300d.png">
</td>
<td>
<img width="178" alt="Screenshot 2020-01-30 at 09 44 26" src="https://user-images.githubusercontent.com/156303/73437933-20c7f080-4345-11ea-993f-b90d45a95d70.png">
</td>
</tr>
</table>

A few thoughts here:

* The angle down icon needs some spacing
* In the design the buttons are of equal width. This to me is an implementation detail so we should adhere to the design but implement the css as if we were using the buttons/dropdowns in a project eg. it's the application's concern to make them equal width not the component we're importing
* The hover/disabled states don't match the design